internal/types/host/keepalived_host_collector_host.go

```diff
@@ -162,7 +162,7 @@ func (k *KeepalivedHostCollectorHost) StatsVrrps() (map[string]*collector.VRRPSt
 	return collector.ParseStats(f)
 }
 
-func (k *KeepalivedHostCollectorHost) DataVrrps() (map[string]*collector.VRRPData, error) {
+func (k *KeepalivedHostCollectorHost) getDataFile() (*os.File, error) {
```

Comment:
By calling this in both functions we will endup with inconsistency - basically, the data file is there - we just got two function to parse the single file after the dump. So I guess the best place for calling this function would be `getKeepalivedStats` before calling the functions for parse. Also, this needs to be taken care of for container struct as well. (probably add the signal func to the interface 🤔 )